script.js

```diff
@@ -38,4 +47,149 @@ $aboutNavMobile.addEventListener('click', () => {
 
 $contactNavMobile.addEventListener('click', () => {
   hideMobileMenu();
-});
\ No newline at end of file
+});
+
+const $worksSection = document.getElementById('works-section');
+
+const ProjectList = [
```

Comment:
- [ ] Good job implementing the array of object to store project info however each object need to contain at least the following information about the project: ```name```, ```description```, ``` featured image```, ```technologies```, ```link to live version```, and ```link to the project source code```, kindly add the link to the project source code and project live version

Reply:
Good job implementing the changes 👍